### ISSUE TITLE
releasing GPKGTileDao resources

### ISSUE THREAD

**Issue by rghosh0** [ngageoint/geopackage-ios#25]
How does one release the resources of a GPKGTileDao object?  The following code leaks memory, but commenting out the tileDao line removes the leak.  But I didn't find a close method in GPKGTileDao or its super-classes.

```
for (int i=0; i<50; i++) {
    gpkg = [_gpkgGeoPackageManager open:geopackageItem.filename];
    GPKGTileDao *tileDao = [gpkg getTileDaoWithTableName:@"DNCBASEMAP12_16"];
    [gpkg close];
}
```

I had a similar experience with GPKGFeatureDao.

**Comment by bosborn** [comment]
Looked through the code and ran some tests.  I haven't determined why this would be occurring. Will revisit when I get more time.

**Comment by rghosh0** [comment]
Here is a fix which apparently solves the problem:

https://github.com/mousebird/geopackage-ios/commit/8a17e43b3103a18808cb77089319b97b4cc09d4f

**Comment by bosborn** [comment]
Thanks, I grabbed it in pull #26 and will get it in the next release.

**Comment by bosborn** [comment]
Released as part of [1.2.1](https://github.com/ngageoint/geopackage-ios/releases/tag/1.2.1)
